### PR TITLE
chore(ci) : Setup dependabot and fix security issues

### PR DESCRIPTION
**Pull request by mmorel-35**
Replaces #209
Concerning gomod updates, I know that you are planning to do it by trivy but as it not active yet, I'm suggesting to do it with Dependabot first.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -16,7 +16,13 @@ jobs:
...
 
     - name: Check out code into the Go module directory
-      uses: actions/checkout@v1 
+      uses: actions/checkout@v1
+    - uses: actions/cache@v2
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
```
Comment (knqyf263) [inline .github/workflows/test.yml:25]: `Install dependencies` takes only 10 seconds. I don't think we need to use cache now.
image

**Review by knqyf263 — COMMENTED** [review]
We are discussing tfsec update in https://github.com/aquasecurity/fanal/pull/229. Please do not update tfsec yet.

File: .github/workflows/test.yml (hunk 1 above)
Comment (mmorel-35) [reply .github/workflows/test.yml]: Done :) !

**Review by knqyf263 — DISMISSED** [review]

**Comment by mmorel-35** [comment]
Please relaunch the Job , this test passes once one time on two

**Comment by mmorel-35** [comment]
It doesn't work on https://github.com/mmorel-35/fanal/actions/runs/1119965762
But it passes on https://github.com/mmorel-35/fanal/actions/runs/1119969448 and I haven't touched this part

**Review by knqyf263 — APPROVED** [review]

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
Thanks

**Comment by knqyf263** [comment]
I couldn't find how to increment only the major version of GitHub Actions. @mmorel-35 Do you know some tips? If no, I'll revert Dependabot.
https://github.com/aquasecurity/fanal/pull/232

**Pull request merged**